tsconfig.packages.json

```diff
@@ -0,0 +1,32 @@
+{
+  "files": [],
+  "references": [
+    {
+      "path": "./modules/account-lib"
+    },
+    {
+      "path": "./modules/bitgo"
+    },
+    {
+      "path": "./modules/blockapis"
```

Comment:
we probably need to rename this package to adhere to our naming convention, possibly `explorer-api`